### DISCUSSION TITLE
Getting the error from Facebook "[190] Error validating access token"

### DISCUSSION THREAD

**Discussion by minhchienwikipedia** [thebergamo/react-native-fbsdk-next#260] [Q&A]
I got this error from Facebook 
`[190] Error validating access token: The session has been invalidated because the user changed their password or Facebook has changed the session for security reasons problem solve`
Does anyone have any solution to fix this issue?

**Comment by mikehardy** [comment]
Sounds like you need to re-authenticate the user.

I imagine it is something like:

Your user: Nice app you have @minhchienwikipedia - sure I will log in using my facebook credentials

Later:

Your user: Oh no! I think my facebook account was hacked! I will go to my security settings and invalidate all sessions in the facebook app

Later:

Your user: Hey I want to see what's going on in @minhchienwikipedia app again, now your app should see that it has an invalid session by that error message, and you should re-authenticate them, possible with a dialog saying "Your session has been closed, please log in again" or similar

**Reply by minhchienwikipedia** [reply]
But this issue shows up when user trying to sign in? Does user need to resolve it by themself?

**Reply by mikehardy** [reply]
You might have mentioned that? You included no information on how your flow went https://stackoverflow.com/help/how-to-ask - I mean, you don't mention versions of software or execution environment + version at all, so it's all guesswork

A generic web search is informative https://search.brave.com/search?q=login+sdk+facebook+%5B190%5D+Error+validating+access+token

seems like logout, then login again might do it again. You share no code, so I can't guess what you tried
